espnet2/asr_transducer/error_calculator.py

```diff
@@ -54,7 +54,10 @@ def __init__(
         self.report_wer = report_wer
 
     def __call__(
-        self, encoder_out: torch.Tensor, target: torch.Tensor
+        self,
+        encoder_out: torch.Tensor,
+        target: torch.Tensor,
+        encoder_out_lens: torch.Tensor,
     ) -> Tuple[Optional[float], Optional[float]]:
         """Calculate sentence-level WER or/and CER score for Transducer model.
```

Comment:
Could you also add the docstring for `encoder_out_lens` to the method, please?

i.e.: `encoder_out_lens: Encoder output sequences length. (B,)`